install/operator/pkg/syndesis/olm/csv.go

```diff
@@ -331,7 +331,7 @@ func (c *csv) build() (err error) {
 			Customresourcedefinitions: CustomResourceDefinitions{
```

Comment:
General note: Definitely want to stop maintaining this olm code. So out of date!

Reply:
Good, I will do this in another PR.